docs/example/DdsImagePlugin.py

```diff
@@ -212,10 +212,10 @@ class DdsImageFile(ImageFile.ImageFile):
     def _open(self):
         magic, header_size = struct.unpack("<II", self.fp.read(8))
         if header_size != 124:
-            raise OSError("Unsupported header size %r" % (header_size))
+            raise OSError(f"Unsupported header size {repr(header_size)}")
```

Comment:
```suggestion
            raise OSError(f"Unsupported header size {header_size!r}")
```

Reply:
Thanks for detailed review!

https://stackoverflow.com/a/44800859/724176 says of `!r`:

> It just calls the `repr` of the value supplied. 
> 
> It's usage is generally not really needed with f-strings since with them you can just do `repr(self.radius)` which is arguably more clear in its intent.
> 
> `!r` (`repr`), `!s` (`str`) and `!a` (`ascii`) were kept around just to ease compatibility with the `str.format` alternative, you don't need to use them with f-strings.

I'm tempted to keep the more explicit version, and the [f-string PEP](https://www.python.org/dev/peps/pep-0498/#s-r-and-a-are-redundant) says "`!s`, `!r`, and `!a` are redundant".

Maybe we should replace the other handful of existing `!r`s with `repr`.

Reply:
> Maybe we should replace the other handful of existing `!r`s with `repr`.

Done in last push.

Reply:
There's one more in `Tests/test_imagemath.py` at the top. I personally prefer `!r` for brevity, but it's fine as long as it's consistent. Perhaps it would be a good idea to add a lint rule for this?